src/types.ts

```diff
@@ -118,7 +118,7 @@ export interface HomeAssistant {
 
   // i18n
   language: string;
```

Comment:
I think that we should rename this to what it represents, or at least mention it in a comment:

it's the locally detected browser language, or, if previously set, a user preference.

And I think that's why our code is a bit complicated: `this.selectedLanguage` becomes `this.language` after a refresh.

Reply:
Add comments, keep `selectedLanguage` for backward compatibility, so that no breaking changes.